<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageConvert</class>
 <widget class="QDialog" name="ImageConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageConvert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt_pco_camera.qrc">
    <normaloff>:/Resources/PCOApp.ico</normaloff>:/Resources/PCOApp.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="GridLeft">
     <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignBottom">
      <widget class="QLabel" name="LabelBottomfarleft">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignTop">
      <widget class="QLabel" name="LabelBottomleft">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="LabelMidleft">
       <property name="text">
        <string>127</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
      <widget class="QLabel" name="LabelTopleft">
       <property name="text">
        <string>255</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" alignment="Qt::AlignRight|Qt::AlignTop">
      <widget class="QLabel" name="LabelBottomright">
       <property name="text">
        <string>65535</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLabel" name="LabelBottomMiddle">
       <property name="text">
        <string>32768</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3" colspan="3">
      <widget class="QGraphicsView" name="ConvertGraphicsView">
       <property name="minimumSize">
        <size>
         <width>229</width>
         <height>145</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>229</width>
         <height>145</height>
        </size>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QGridLayout" name="GridRight">
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="GammaLabel">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="BottomValueBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSpinBox" name="TopValueBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="Spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="GammaDoubleBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_pco_camera.qrc"/>
 </resources>
 <connections/>
</ui>
